shell/platform/android/flutter_main.cc

```diff
@@ -77,6 +77,54 @@ const flutter::Settings& FlutterMain::GetSettings() const {
   return settings_;
 }
 
+// TODO: Move this out into a separate file?
```

Comment:
Yeah moving this into a separate file would definitely make it easier to keep up with engine updates